src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -36,7 +36,7 @@ class HuiEntitiesCard extends LitElement implements LovelaceCard {
   }
 
   public static getStubConfig(): object {
-    return { entities: [] };
+    return { entities: [], state_color: false };
```

Comment:
Don't add it to the stub, it is not a required setting

Reply:
If you don't then the editor shows it as true on the creation of a new one